style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  font-size: 50px;
+}
```

Comment:
- [ **OPTIONAL** ] I think it would be better to style your heading by using a **class** as required for the project. This should be pointed out by the previous reviewer. I know you can do it :muscle:.

Reply:
Thank you [@tresorsawasawa](https://github.com/tresorsawasawa) I will check on that